### ISSUE TITLE
Playing game: xyz

### ISSUE THREAD

**Issue by axel1200** [FAForever/downlords-faf-client#501]
![grafik](https://cloud.githubusercontent.com/assets/19996713/23041415/342e8208-f495-11e6-8920-c65b3f89376c.png)
Look at axel12... he seems to be playing "pure skill<500"
Regarding my ranking I did never play that game.

**Comment by axel1200** [comment]
Wait maybe that was because I was logged in on test Server :D

**Comment by Geosearchef** [comment]
I can confirm that there currently is a bug in the client (already solved in branch #182) which causes it to not recognize a player leaving a game.
But this here seems to me as being a bug in the server. I'm always displayed as still being in a game I played some time ago (sometimes switches). Maybe this caueses you being displayed as in a game you never played.

I traced this bug in the client and came to the conclusion that the misinformation is sent by the server (also happens in the python client). I remember creating an issue in the server repository but can't manage to find it right now.
